Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -415,6 +415,15 @@ class BrowserViewController: UIViewController {
             // Reset previous crash state
             activeCrashReporter?.resetPreviousCrashState()
 
+            // Only ask to restore tabs from a crash if we had non-home tabs or tabs with some kind of history in them
+            guard let tabsToRestore = tabManager.tabsToRestore() else { return }
+            let onlyNoHistoryTabs = !tabsToRestore.every { $0.sessionData?.urls.count > 1 }
+            if onlyNoHistoryTabs {
+                let tab = tabManager.addTab()
+                tabManager.selectTab(tab)
```

Comment:
Further to the other crash reporter bug: it seems like `TabManager` should be in charge of saying "huh, guess I have no tabs; maybe I'll open a blank one and select it".